src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div key={movie.title}>
+        <MovieCard {...movie} />
+      </div>
```

Comment:
```suggestion
      <MovieCard key={movie.imdbId} movie={movie} />
```

Reply:
you can set `key `directly to the component. `imdbId `is unique, but `title `can repeat in different movies. decompose movie inside the component